app/javascript/runtimeBanner/RuntimeBanner.jsx

```diff
@@ -6,6 +6,10 @@ import CloseIcon from '@images/x.svg';
 import LogoForem from '@images/logo-forem-app.svg';
 
 const BANNER_DISMISS_KEY = 'runtimeBannerDismissed';
+const APP_LAUNCH_SCHEME = 'com.forem.app';
+const FOREM_APP_STORE_URL = 'https://apps.apple.com/us/app/forem/id1536933197';
+const FOREM_GOOGLE_PLAY_URL =
+  'https://play.google.com/store/apps/details?id=com.forem.android';
```

Comment:
Refactor (suggestion): These can be environment variables that webpack can inject if it makes sense. If these never/rarely change, keeping them as constants is fine.

Reply:
Thanks for the suggestion @nickytonline! Yup these AppStore/Play Store links aren't supposed to ever change so we should be safe with this approach then 👍